assets/css/tasks/_article-tasks.css

```diff
@@ -0,0 +1,114 @@
+/* SINGLE ARTICLE PAGE */
+.container {
+    font-family: 'Lato', sans-serif;
+}
+
+.post-page__img { 
+    width: 100%;
+    max-height: 250px;
+    object-fit: contain;
+    overflow: hidden;
+    object-fit: cover;
+    padding: 0px 250px 0px 250px;
+    margin-top: 50px;
+}
+
+.post-page__header {
+    width: 100%;
+    padding-bottom: 100px;
+}
+
+.u-margin--medium {
+    margin: 100px 350px 50px 350px;
+}
+
+.post-description__title {
+    float: left;
+    margin-bottom: 30px;
+}
+
+.post-description__date {
+    float: right;
+}
+
+.post-end {
+    margin: 100px 300px 50px 300px;
+}
+
+.post-page-end__button--previous,
+.post-page__author-info,
+.post-page-end__button--next {
+    float: left;
+    width: 33%;
+    /* width; all three elements at the end of the page should share the available width between them, so one element should take up 1/3 of the entire available width*/
+}
+
+.post-page-end__button--next {
+    float: right;
+    color: #7a24ab;
+    text-align: right;
+    font-weight: bold;
+    padding-top: 35px;
+}
+
+.post-page-end__button--previous { 
+    text-align: left; 
+    color: #7a24ab;
+    font-weight: bold;
+    padding-top: 35px;
+}
+
+.arrow {
+    border: solid #7a24ab;
+    border-width: 0 3px 3px 0;
+    display: inline-block;
+    padding: 3px;
+  }
+  
+.right {
+    transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
+  }
+  
+.left {
+    transform: rotate(135deg);
+    -webkit-transform: rotate(135deg);
+  }
+
+.post-page-end__button--next:hover, .post-page-end__button--previous:hover {
+    cursor: pointer;
+    font-weight: lighter;
+}
+
+.author-box__img {
+    text-align: center;
+    float: left;
+    object-fit: contain;
+    border-radius: 50%; 
+    width:100px;
+    height:100px;
+    background-color: antiquewhite;
+}
+
+
+.author-box__description {
+    text-align: left;
+    padding-left: 120px;
+}
+
+.author-description {
+   text-align: center;
+}
+
+.author-title {
+    font-weight: lighter;
+}
+
+.author-title {
+    margin-top: 15px;
+}
+
+.author-info--name {
+    font-weight: bold;
+    margin: 5px 0px 5px 0px;
```

Comment:
This is good.
I'm glad you are using the shorter version for adding values ​​for the margin properties. There is another way to deal with this type of margin formation. Even shorter! :)
```
margin: 5px 0;
```
If you have the same top and bottom margin values ​​and the same right and left values, you can just set one vertical and one horizontal value.

A little tip. If you want a property's value to be zero, you don't need to set the unit to zero.